output/eu_crude/analysis.py

```diff
@@ -0,0 +1,153 @@
+import sqlite3 as lite
+import sys
+import matplotlib as plt
+from pyne import nucname
+
+# tells command format if input is invalid
+if len(sys.argv) < 2:
+    print('Usage: python analysis.py [cylus_output_file]')
+
+
+def analysis(filename):
```

Comment:
I would call this something like "snf". A function called "analysis" promises too much. A function should do exactly one thing (http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745)